test/testcafe/spec/WidgetCustomization_spec.js

```diff
@@ -65,69 +65,72 @@ async function setupPasswordExpired(t) {
 }
 
 
-test.requestHooks(identifyMock)('should show custom footer links', async t => {
```

Comment:
these custom link tests will be enabled once this PR is merged: https://github.com/okta/okta-signin-widget/pull/3236